2021/46xxx/CVE-2021-46380.json

```diff
@@ -1,8 +1,8 @@
 {
```

Comment:
{} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'version': {'version_data': [{}]}}]}}]}}